### PR TITLE
Documentation update PRO-100

### PR DESCRIPTION
**Pull request by LouS22**

### CODE REVIEW

File: docs/value(s).md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# all about value(s)
```
Comment (smakhtin) [inline docs/value(s).md:1]: Should this be capitalized?

File: docs/value(s).md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## **value at the heart of the conversation**
```
Comment (smakhtin) [inline docs/value(s).md:3]: Same here, should we capitalize this?

File: docs/value(s).md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+We counteract this paradigm by developing new ways of valuing the work of ecological stewards, moving the appreciation from 'nice to have’ toward ‘essential necessity'. The key to this shift lies in integrating the eco-data the steward produce into products of the digital economy thereby making it indispensable. This differs from conventional approaches, usually in the realm of what has come to be known as eco-system services. In our proposition, the value derives from the utility of the data that is produced by doing ecological work. It is thus independent from typical valuations of nature as natural capital in service of human life. The commodification is located at the level of data production and data utility, leaving the integrity of ecological ground work intact. 
+
+Our valuation practices are not aimed at putting a price tag on nature, or assess the value of its eco-systemic services. We are building a new economy in the digital world that leaves enough on-site space for ecological stewards to continue doing their work, not having to succumb to the pressures of the market while stepping into the digital economy. 
+
+## **digital life not digital twinning**
```
Comment (smakhtin) [inline docs/value(s).md:11]: Please check header formatting

File: docs/value(s).md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Our pipeline connects eco-data to the digital creative space, where individuals and communities create representations and assets, so-called eco-derivatives. By utilising our protocol, these digital creatives can access and incorporate eco-data provided by ecological stewards into their projects. We do not impose a specific aesthetic or representation of nature in the digital world, but rather empower digital creatives to collaborate, innovate, and showcase their unique perspectives. We embrace the idea that there must be multiple valid representations of eco-data, multiplying the impact and possibilities for ecological work.
+
+## **Sovereign Nature Initiative**
```
Comment (smakhtin) [inline docs/value(s).md:15]: Probably wrong title

**Review by smakhtin — COMMENTED** [review]
@LouS22 Looks good to me, please check my comments considering the header formatting.

It would be also helpful if you can draft sidebar structure - where should we put new pages?

File: docs/value(s).md (hunk 1 above)
Comment (LouS22) [reply docs/value(s).md:1]: no, should stay small

Comment (LouS22) [reply docs/value(s).md:1]: or rather, capitalising the first letter - All about value(s)

File: docs/value(s).md (hunk 2 above)
Comment (LouS22) [reply docs/value(s).md:3]: same here, just the first word

**Review by smakhtin — APPROVED** [review]

**Pull request merged**